examples/02c_wind_farm_realistic_inflow_direct/hercules_input.yaml

```diff
@@ -0,0 +1,42 @@
+# Input YAML for hercules
+
+# Name
+name: example_02c
+
+###
+# Describe this simulation setup
+description: Wind Only Realistic Inflow (Direct - No Wake Modeling)
+
+dt: 1.0
+starttime_utc: "2024-06-24T16:59:08Z" # Jun 24, 2024 16:59:08 UTC (Zulu time)
+endtime_utc: "2024-06-26T16:59:00Z" # ≈48 hours later (Jun 26, 2024 16:59:00 UTC)
+verbose: False
+
+plant:
+  interconnect_limit: 45000 # kW
+
+wind_farm:
+  
+  component_type: Wind_MesoToPowerNoAddedWakes
+  floris_input_file: ../inputs/floris_input_large.yaml
+  wind_input_filename: ../inputs/wind_input_large.ftr
+  turbine_file_name: ../inputs/turbine_filter_model.yaml
+  log_file_name: outputs/log_wind_sim.log
+  log_channels:
+    - power
+    - wind_speed_mean_background
+    - wind_speed_mean_withwakes
+    - wind_direction_mean
+    - turbine_powers
+    - wind_speeds_withwakes
+    - wind_speeds_background
+    - turbine_power_setpoints
+  floris_update_time_s: 300.0 # Not used but kept for interface consistency
```

Comment:
Is there an easy way to remove this for better clarity on what's happening?

Reply:
This is now done.